package.json

```diff
@@ -12,12 +12,16 @@
     "readline": "^1.3.0"
   },
   "devDependencies": {
+    "coveralls": "^2.13.1",
     "eslint": "^3.19.0",
+    "istanbul": "^0.4.5",
     "jasmine": "^2.6.0"
   },
   "scripts": {
     "lint": "./node_modules/.bin/eslint .",
     "lint-fix": "./node_modules/.bin/eslint --fix .",
-    "test": "./node_modules/.bin/jasmine"
+    "test": "./node_modules/.bin/jasmine",
+    "coverage": "./node_modules/istanbul/lib/cli.js cover ./node_modules/jasmine/bin/jasmine.js",
+    "report-coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
```

Comment:
I think you should remove this script and append the command to the last one using `&&` as shown in https://github.com/nickmerwin/node-coveralls#istanbul